CHANGELOG.md

```diff
@@ -10,6 +10,7 @@ Versioning](https://semver.org/spec/v2.0.0.html).
 
 ### Added
 
+- Add auto deployment of main in circle-ci
```

Comment:
As this work does not impract production code, add an entry into the CHANGELOG is not recommended.